README.md

```diff
@@ -277,7 +277,7 @@ to match yours.
 
     <filter>
         <filter-name>SAML2SSOAgentFilter</filter-name>
-        <filter-class>org.wso2.carbon.identity.sso.tomcat.server.SAML2SSOAgentFilter</filter-class>
+        <filter-class>SAML2SSOAgentFilter</filter-class>
```

Comment:
Whats the best practice to refer filter class name? With or without FQN

Reply:
Prepended FQN in https://github.com/gayashanbc/identity-agent-sso/pull/1/commits/3ee12f935a59ce79bdb211aebcaaf7213a262327